### PR TITLE
feat: arbitrable

### PR DESCRIPTION
**Pull request by ashitakah**
# 🤖 Linear

Closes GRT-148

### CODE REVIEW

File: test/unit/CouncilArbitrator.t.sol [hunk 1]
```diff
@@ -40,8 +39,7 @@ contract CouncilArbitrator_Unit_BaseTest is Test {
 
   IOracle public oracle;
   IArbitratorModule public arbitratorModule;
-  address public arbitrator;
-  address public council;
+  IArbitrable public arbitrable;
 
   event ResolutionStarted(
     bytes32 indexed _disputeId, IOracle.Request _request, IOracle.Response _response, IOracle.Dispute _dispute
```
Comment (0xJabberwock) [inline test/unit/CouncilArbitrator.t.sol:45]: Erase `SetArbitrator` and `SetCouncil` events.

File: src/contracts/Arbitrable.sol [hunk 2]
```diff
@@ -7,19 +7,11 @@ import {IArbitrable} from 'interfaces/IArbitrable.sol';
  * @title Arbitrable
  * @notice Makes a contract subject to arbitration by The Graph
  */
-abstract contract Arbitrable is IArbitrable {
+contract Arbitrable is IArbitrable {
```
Comment (0xJabberwock) [inline src/contracts/Arbitrable.sol:10]: I think the `@notice` and `@title` should be updated. I propose `ArbitrationHub` for the latter.

**Review by 0xJabberwock — CHANGES_REQUESTED** [review]
Exquisite PR!

File: src/contracts/Arbitrable.sol [hunk 3]
```diff
@@ -61,6 +53,11 @@ abstract contract Arbitrable is IArbitrable {
     __pendingCouncil = _pendingCouncil;
   }
 
+  /// @inheritdoc IArbitrable
+  function isValidArbitrator(address _caller) public view {
```
Comment (0xShaito) [inline src/contracts/Arbitrable.sol:57]: Would prefer validateArbitrator

File: test/unit/CouncilArbitrator.t.sol [hunk 4]
```diff
@@ -178,19 +158,15 @@ contract CouncilArbitrator_Unit_ResolveDispute is CouncilArbitrator_Unit_BaseTes
     _params.answer = uint8(IOracle.DisputeStatus.None);
 
     councilArbitrator.mock_setResolutions(_params.disputeId, _params.resolutionData);
-
-    vm.startPrank(arbitrator);
+    vm.mockCall(
```
Comment (0xShaito) [inline test/unit/CouncilArbitrator.t.sol:161]: mockAndExpect pls

File: src/contracts/EBORequestCreator.sol [hunk 5]
```diff
@@ -11,16 +11,17 @@ import {IBondedResponseModule} from
...
 import {IEBORequestModule} from 'interfaces/IEBORequestModule.sol';
 
-contract EBORequestCreator is Arbitrable, IEBORequestCreator {
+contract EBORequestCreator is IEBORequestCreator {
   using EnumerableSet for EnumerableSet.Bytes32Set;
 
   /// @inheritdoc IEBORequestCreator
   IOracle public immutable ORACLE;
 
+  IArbitrable public immutable ARBITRABLE;
```
[the remark this answers is not in the document]
Comment (ashitakah) [reply src/contracts/EBORequestCreator.sol:23]: 👍

File: test/unit/Arbitrable.t.sol [hunk 6]
```diff
@@ -11,7 +11,9 @@ contract MockArbitrable is Arbitrable {
   constructor(address _arbitrator, address _council) Arbitrable(_arbitrator, _council) {}
 
   // solhint-disable-next-line no-empty-blocks
-  function mock_onlyArbitrator() external onlyArbitrator {}
+  function mock_onlyArbitrator() external {
+    isValidArbitrator(msg.sender);
```
[the remark this answers is not in the document]
Comment (ashitakah) [reply test/unit/Arbitrable.t.sol:15]: 👍

File: test/unit/CouncilArbitrator.t.sol (hunk 1 above)
Comment (ashitakah) [reply test/unit/CouncilArbitrator.t.sol:45]: 👍

File: test/unit/CouncilArbitrator.t.sol (hunk 4 above)
Comment (ashitakah) [reply test/unit/CouncilArbitrator.t.sol]: 👍

**Review by 0xShaito — APPROVED** [review]

**Review by 0xJabberwock — APPROVED** [review]

**Pull request merged**